apps/aefate/src/aefa_fate_op.erl

```diff
@@ -2675,6 +2764,12 @@ op(Op, Arg1, Arg2) ->
     aefa_fate:abort({type_error, Op, [Arg1, Arg2]}).
 
 %% Terinay operations
```

Comment:
*ternary

Reply:
I'm pretty sure I didn't write that 🤔 

Ah, not part of the change, but yes correct.